Makefile

```diff
@@ -100,6 +100,13 @@ dev-link:
 	cd node_modules/@elyra/pipeline-editor && jupyter labextension link --no-build .
 	cd node_modules/@elyra/pipeline-services && jupyter labextension link --no-build .
 
+dev-unlink:
+	yarn unlink @elyra/pipeline-services
+	yarn unlink @elyra/pipeline-editor
+	jupyter labextension uninstall @elyra/pipeline-services
+	jupyter labextension uninstall @elyra/pipeline-editor
```

Comment:
this is handled in `uninstall`, is there a reason we need another command?

Reply:
I think it's convenient to have a command to just unlink, but if others don't find it useful I can remove

Reply:
I think that'd be useful

Reply:
fair enough I guess make sure to add `dev-unlink` to `.PHONY` since it's a new command

Reply:
oh yea I think we also need to add dev-link to .PHONY too

Reply:
thx, I had added it, but I must've been lost in some merge or rebase along the way